src/components/SwitchCheckboxIcon/SwitchCheckboxIcon.scss

```diff
@@ -1,67 +1,314 @@
 @import 'foundation/colors';
 .switchCheckbox {
-    transition: all .3s ease-in-out;
-    -moz-transition: all .3s ease-in-out;
-    -webkit-transition: all .3s ease-in-out;
-    .OuterSpan {
+    display: flex;
+    .switchLabel {
+        margin-left:6px;
+    }
+    .outeWrap {
+        display: flex;
+        justify-content: space-between;
+        align-items: flex-start;
+        width: 54px;
+        border: 1px solid;
+        border-radius: 30px;
+        background: transparent;
         position: relative;
-        margin-right: 0.8rem;
-        .InnerSpan {
-            width: 4.4rem;
-            background:color(ink, lighter);
-            display: inline-block;
+        border:1px solid #000;
+        height:22px;
+        & > label {
+            width: 33.33%;
             position: relative;
-            border-radius: 3rem;
-            height: 2.2rem;
-            vertical-align: middle;
-            .innerCircle{
-                content: "";
-                display: block;
-                width: 1.9rem;
-                height: 1.9rem;
-                border-radius: 100%;
-                position: absolute;
-                left: 1px;
-                right:auto;
-                background-color: color(sky, base);
-                top: 1px;
-                transition: all .3s ease-in-out;
-                -moz-transition: all .3s ease-in-out;
-                -webkit-transition: all .3s ease-in-out;
-                z-index: 1;
-                .Accept {
-                    display: none;
-                }
-                .Cancel {
-                    display: block;
-                }
-            }
+            top: 0px;
+            left: 0px;
+        }
+        .switchRadio {
+            font-size: 0px;
+            color:transparent;
+            width:19px;
+            height:19px;
+            background:#fff;
```

Comment:
This should use theme color (this applies everywhere). Take a look at https://github.com/emgage/engage-ui/blob/master/src/components/TextField/TextField.scss and https://github.com/emgage/engage-ui/blob/master/themes/Delicious/TextField.scss to see how this should be done for TextField and then follow that model. HEX color values should not be used in most cases.

Also, this commit is missing the Delicious theme styling for this component.